compiled_starters/ruby/app/main.rb

```diff
@@ -22,7 +22,7 @@ def decode_bencode(bencoded_value)
 
 if command == "decode"
     # You can use print statements as follows for debugging, they'll be visible when running tests.
-    puts "Logs from your program will appear here"
+    warn "Logs from your program will appear here"
```

Comment:
In Ruby, I think the more common way to do this would be `$stderr.puts("...")`. `warn` isn't used as much + it's often used specifically for warnings